.github/tests/javascript/graphql-sdl-first/run.sh

```diff
@@ -2,13 +2,14 @@
 
 set -eu
 
-yarn 
-yarn prisma db push --preview-feature
-yarn dev &
+npm install
+npx prisma migrate dev --name init
+npx prisma db seed --preview-feature
+npm run dev &
 pid=$!
 
-sleep 10
+sleep 20
 
-curl --fail 'http://localhost:4000/' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: http://localhost:4000' --data-binary '{"query":"query {\n  feed {\n    id\n    content\n    author {\n      id\n      name\n      email\n    }\n  }\n}"}' --compressed
+npx newman run https://raw.githubusercontent.com/prisma/prisma-examples/latest/.github/tests/postman_collections/graphql.json --bail
```

Comment:
Can we not run this with a path instead of an absolute URL?

Reply:
I tried that; it didn't work.

Each test is only "aware" of the example and can't access any resource outside the example.

Reply:
That does not sound correct to me. You should be able to test the examples locally by running the local tests. Can we not just supply a relative path?

Reply:
I tried using a relative path in this [Action's run](https://github.com/prisma/prisma-examples/runs/2256064637?check_suite_focus=true#step:5:115), it couldn't find the file.

![image](https://user-images.githubusercontent.com/33921841/115071217-a4e0a680-9efe-11eb-846d-c98f6f304a89.png)

Reply:
The script currently probably runs in the project folder context, so `/javascript/rest-express`. So paths would need to be relative to that or change that context first.

Reply:
That would lead to some redundancy, especially for the graphql examples. I opted for a central location for all the tests.

Reply:
No, you can still leave the files where they are. But you reference them with a relative path to where the script is run (as the script is not run in the folder where it lives, but in the project folder).

Example could be: 
```suggestion
npx newman run ../../.github/tests/postman_collections/graphql.json --bail
```

That of course means they will not run if you execute `.github/tests/javascript/graphql-sdl-first/run.sh` directly any more, but let's deal with this later after we confirmed the relative path works on CI.

Reply:
Sorry, I misunderstood that.

Reply:
That worked! I'll update the other tests